backend/census_historical_migration/workbooklib/secondary_auditors.py

```diff
@@ -155,6 +155,30 @@ def xform_cpafirmname(secondary_auditors):
         InspectionRecord.append_secondary_auditor_changes(change_records)
 
 
+def xform_pad_contact_phone_with_nine(secondary_auditors):
+    """Pad contact phone with 9s if less than 10 digits"""
+    change_records = []
+    is_pad_applied = False
+    for secondary_auditor in secondary_auditors:
+        contact_phone = string_to_string(secondary_auditor.CPAPHONE)
+        if len(contact_phone) < 10 and contact_phone == "999999999":
+            contact_phone = contact_phone.ljust(10, "9")
```

Comment:
```suggestion
        if contact_phone == "999999999":
            contact_phone = "9999999999"
```
I think this is equivalent?